### PR TITLE
CNV-96317: add e2e test for VM search empty state format

### PR DESCRIPTION
**Pull request by adamviktora**
## 📝 Description

add e2e test for VM search empty state format

## 🔗 Links

Jira: https://redhat.atlassian.net/browse/CNV-96317

## 🎥 Demo

N/A

## Summary by CodeRabbit

* **Bug Fixes**
  * Corrected the virtual machine search empty state to display the complete search query, including combined status filters and excluded operating systems.
  * Empty results now clearly show queries such as `status:Paused -os:RHEL`, making it easier to understand why no matches were found.

* **Tests**
  * Added automated coverage to verify empty-state query formatting for mixed inclusion and exclusion filters.

### CODE REVIEW

File: playwright/tests/gating/virtualmachines/vm-search/vm-search-language.spec.md [hunk 1]
```diff
@@ -294,25 +297,43 @@ that submitted queries actually show and hide VirtualMachines in the list.
 
 ---
 
+### `016`: Empty state text formats exclusion filters using search language syntax
+
+- **Objective:** Verify that when a submitted query matches zero VMs, the "No results found for ..."
+  empty state renders every token using search-language syntax (`key:value`, exclusions prefixed with
+  `-`) instead of the pre-fix raw filter dump (e.g. `os:!RHEL` or comma-joined values).
+- **Target version:** CNV 5.0.0
+- **Jira References:** CNV-96225
```
[the remark this answers is not in the document]
Comment (galkremer1) [reply playwright/tests/gating/virtualmachines/vm-search/vm-search-language.spec.md:306]: Yep - target version should be v5.1.0

**Comment by galkremer1** [comment]
/test-e2e playwright/tests/gating/virtualmachines/vm-search/vm-search-language.spec.ts

File: playwright/tests/gating/virtualmachines/vm-search/vm-search-language.spec.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adamviktora) [reply playwright/tests/gating/virtualmachines/vm-search/vm-search-language.spec.md]: I think Jira reference should stay `CNV-96225`, because that is the ticket for the bug this test is testing

**Comment by upalatucci** [comment]
/test-e2e playwright/tests/gating/virtualmachines/vm-search/vm-search-language.spec.ts

**Review by upalatucci — APPROVED** [review]
